apiserver/facades/agent/machine/package_test.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2013 Canonical Ltd.
+// Copyright 2024444 Canonical Ltd.
```

Comment:
Still got this odd one.

Reply:
I'm almost certain I had fixed this and many of the other comments in this second pass :thinking: . This smells like a bad rebase...